UI/Pages/TuitionPartner.cshtml.cs

```diff
@@ -79,9 +79,9 @@ public async Task<IActionResult> OnPostRemoveFromShortlist([FromBody] string seo
     private ArgumentException GetArgumentException(string name) => new($"{name} is null or whitespace");
     private JsonResult GetShortlistJsonResult(bool status) => new(new { Updated = status });
 
-    private IActionResult ReturnNotFound(string logMessage)
+    private IActionResult ReturnNotFoundWithLogging(string logMessage, LogLevel logLevel)
     {
-        _logger.LogWarning("{LogMessage}", logMessage);
+        _logger.Log(logLevel, "{LogMessage}", logMessage);
```

Comment:
Not sure we need to pass in the log level, as it's only ever passed in as `LogLevel.Information`, so you could have just updated this line to
```suggestion
        _logger.LogInformation("{LogMessage}", logMessage);
```

Reply:
If we make the method more flexible, we could reuse it for any other log level. That was my thought process. Happy to apply your suggested approach, as well.

Reply:
TBH this whole `ReturnNotFound` method is pretty superfluous anyway, I reckon!

Reply:
Yeah, agree.